src/Pay/Pay.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Utopia\Pay;
+
+class Pay
+{
+  /**
+   * @var Adapter
+   */
+  protected Adapter $adapter;
+
+  public function __construct(Adapter $adapter)
+  {
+    $this->adapter = $adapter;
+  }
+
+  public function setTestMode(bool $testMode)
+  {
+    return $this->adapter->setTestMode($testMode);
+  }
+
+  public function getTestMode(): bool
+  {
+    return $this->adapter->getTestMode();
+  }
+
+  /**
+   * Get name of the payment gateway
+   */
+  public function getName(): string
+  {
+    return $this->adapter->getName();
+  }
+
+  /**
+   * Set the currency for payments
+   */
+  public function setCurrency(string $currency)
+  {
+    return $this->adapter->setCurrency($currency);
+  }
+
+  /**
+   * Get currently set currency for payments
+   */
+  public function getCurrency(): string
+  {
+    return $this->adapter->getCurrency();
+  }
+
+  /**
+   * Make a purchase request
+   * Returns payment ID on successfull payment
+   * 
+   */
+  public function purchase(float $amount, string $customerId, string $cardId = null, array $additionalParams = []): array
+  {
+    return $this->adapter->purchase($amount, $customerId, $cardId, $additionalParams);
+  }
+
+  /**
+   * Refund payment
+   */
+  public function refund(string $paymentId, float $amount): array
+  {
+    return $this->adapter->refund($paymentId, $amount);
+  }
+
+  /**
+   * Cancel payment
+   */
+  public function cancel(string $paymentId): bool
+  {
+    return $this->adapter->cancel($paymentId);
+  }
+
+  /**
+   * Delete a credit card record
+   */
+  public function deleteCard(string $customerId, string $cardId): bool
+  {
+    return $this->adapter->deleteCard($customerId, $cardId);
+  }
+
+
+  /**
+   * Add a credit card for a customer
+   */
+
+  public function createCard(string $customerId, string $cardId): array
+  {
+    return $this->adapter->createCard($customerId, $cardId);
+  }
+
+  /**
+   * Update credit card
+   */
+
+  public function updateCard(string $customerId, string $cardId, $name = null,  $expMonth = null, $expYear = null, $billingDetails = null): array
+  {
+    return $this->adapter->updateCard($customerId, $cardId, $name, $expMonth, $expYear, $billingDetails);
+  }
+
+  /**
+   * Get credit card
+   */
+  public function getCard(string $customerId, string $cardId): array
+  {
+    return $this->adapter->getCard($customerId, $cardId);
+  }
+
+  /**
+   * List cards
+   */
+  public function listCards(string $customerId): array
+  {
+    return $this->adapter->listCards($customerId);
+  }
+
+  /**
+   * List customers
+   */
+  public function listCustomers(): array {
```

Comment:
```suggestion
  public function listCustomers(): array 
  { 
```

Lets be consistent with the placement of the curly braces as well.